Makefile

```diff
@@ -96,6 +96,12 @@ pydocstyle:
 doc-show:
 	open docs/_build/html/index.html
 
+docs-all:
+	which python
+	python setup.py install --user
```

Comment:
This error happens because you `import gammapy` from the source directory, but the version there isn't built.
Change `python setup.py install --user` to something that does an in-place build, e.g. `python setup.py build_ext -i` or I'd put this: `pip install -e .`

https://travis-ci.org/gammapy/gammapy/jobs/429778098#L2476

It's similar for docs build and testing, you can either build and import Gammapy in-place (that's what `pytest gammapy` does) or build and put a copy somewhere else and use it (that's what `python setup.py build_docs` or `python setup.py test` do). The easiest for `process_tutorials.py` is to do a Gammapy in-place build first.